packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -782,6 +817,25 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
         }
     }
 
+    private renderMeasureSpan(ref: React.RefObject<HTMLSpanElement>, displayString: string): JSX.Element {
```

Comment:
please be more verbose on method name so on first glance a reader can understand what it is for. It can be quite long as this all gets minified for production.

Reply:
I suggest since this is measuring the display names we call it renderSelectTextStringsForMeasurement and add an additional comment above "Renders a span to measure the display names to make sure the width of the select is consistent"

Reply:
I'd recommend reversing the naming pattern.  In API's they are easier to understand and learn when things can group together by name.  So if you started with "Measure..." anything related to measuring would show up together in an alphabetized list of function names. 

Either way, always document with function comments, name, description, inputs, outputs.  this will then ensure the name of the function, whatever it is, has a good description for verbosity and examples are nice as well.